ui/src/components/SampleGrid/SampleGridTable.css

```diff
@@ -6,6 +6,10 @@
   display: inline;
 }
 
+.samples-grid-table {
+  flex-wrap: nowrap;
```

Comment:
I think users prefer scrolling vertical than horizontal.
I would suggest to remove this style flex-wrap: nowrap; ?